student.rb

```diff
@@ -1,4 +1,4 @@
-require_relative person
+require_relative 'person'
 
```

Comment:
- [ ] One of the requirements here is that when setting the `classroom` for a `student` it also adds it to the classrooms' students: Which means you need to set a getter for the classroom class which you did not create initially. Kindly add the `getter 👍 

### Instructions:
![image](https://github.com/akezeth/OOP-School-Library/assets/50990925/2acfba22-9040-412d-9619-c3564d083dd0)